### ISSUE TITLE
Why test accuracy reaches high as 90% while the train accuracy is about 50%?

### ISSUE THREAD

**Issue by Arthurfangm** [winycg/HCGNet#2]
Hello, thanks for sharing your work. When I run your code on my machine with cifar10 dataset, I notice that the test accuracy is already as high as 90%, while the train accuracy is still around 50%. In my experience, training accuracy is always higher than the test accuracy, so I have a question "Why test accuracy reaches high as 90% while the train accuracy is about 50%?"  Thanks again for sharing your excellent work!

**Comment by winycg** [comment]
Thanks for your attention! The reason of the low training accuracy derives from the regularizations of mixup and label smooth for preventing overfitting.  Moreover, the statistics of training accuracy based on mixed images, which would not reflect actual   accuracy of training set in the training stage. So if you would like to explore the actual accuracy  of training set, you should evaluate unmixed training images at training stage. Thanks!

**Comment by Arthurfangm** [comment]
Thanks for replying. That reminds me again how powerful regularization is.  Thanks !
